frontend-ssr/services/LessonComments.service.ts

```diff
@@ -0,0 +1,60 @@
+import { uuid } from '~/services/Utils';
+import { Comment } from '../redux/lesson-comments/types';
+
+const MOCK_COMMENTS: Comment[] = [
+  {
+    _id: uuid(),
+    username: 'Diana123',
+    name: 'Diana',
+    avatar: 'https://www.gravatar.com/avatar/9d4d790eb85e56d6524faed8a883d6cb?s=500&d=404',
+    comment: 'Lorem ipsum sim dolore',
+    timestamp: Date.now(),
+  },
+  {
+    _id: uuid(),
+    username: 'AlexPava',
+    name: 'Alex',
+    avatar: 'https://joeschmoe.io/api/v1/josh',
+    comment: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry',
+    timestamp: Date.now(),
+  },
+  {
+    _id: uuid(),
+    username: 'Andrei82',
+    name: 'Andrei',
+    avatar: 'https://joeschmoe.io/api/v1/jon',
+    comment: 'Why do we use it?',
+    timestamp: Date.now(),
+  },
+  {
+    _id: uuid(),
+    username: 'AntoniaBebe',
+    name: 'Antonia',
+    avatar: 'https://joeschmoe.io/api/v1/jess',
+    comment: 'What is Lorem Ipsum?',
+    timestamp: Date.now(),
+  },
+  {
+    _id: uuid(),
+    username: 'Raluca99',
+    name: 'Raluca',
+    avatar: 'https://joeschmoe.io/api/v1/jane',
+    comment: 'Where does it come from?',
+    timestamp: Date.now(),
+  },
+];
+
+const LOAD_DURATION = 1000;
+
+class LessonCommentsService {
+  static wait(mili) {
+    return new Promise((resolve) => setTimeout(resolve, mili));
+  }
+
+  static async getComments(id) {
```

Comment:
You should have another method here for adding a comment. Then in the component we "wait" for it to finish before dispatching the new comment.